src/extension.ts

```diff
@@ -969,7 +974,6 @@ export async function activate(context: vscode.ExtensionContext): Promise<any> {
       )
     ),
     vscode.window.onDidChangeActiveTextEditor((editor: vscode.TextEditor) => {
```

Comment:
Suggest you use a `const` for `editor.document.uri.toString()` in place of the two occurrences below.